### PR TITLE
fix: add titles to sidebar's nodes

### LINKED ISSUE #242: Tooltip to sidebar icons
#### Description
Please try answering a few of those questions
- What changes have to be introduced?
   - Add tooltip to sidebar icons and visualizer icons
- Will this be a breaking change?
   - No 

**Screenshot**

image

### PR DESCRIPTION
**Pull request by Samridhi-98**
**Description**

-  Added tooltip to `sidebar.tsx`
-  Added tooltip to `settings.tsx`

**Related issue(s)**

Fixes #242

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@magicmatatjahu please review my PR.

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -101,12 +101,12 @@ export const Sidebar: React.FunctionComponent<SidebarProps> = () => {
...
             onClick={() => setActiveNav(item.name as NavItemType)}
-            className={`flex text-sm border-l-2  ${
-              item.state()
-                ? 'text-white hover:text-gray-500 border-white'
-                : 'text-gray-500 hover:text-white border-gray-800'
-            } focus:outline-none border-box p-4`}
+            className={`flex text-sm border-l-2  ${item.state()
+              ? 'text-white hover:text-gray-500 border-white'
+              : 'text-gray-500 hover:text-white border-gray-800'
+              } focus:outline-none border-box p-4`}
```
Comment (magicmatatjahu) [inline src/components/Sidebar.tsx:109]: That change will (probably) cause problems on the eslint side, so please run `npm run lint:fix` script.

Comment (magicmatatjahu) [reply src/components/Sidebar.tsx:109]: Yep, it causes errors -> https://github.com/asyncapi/studio/runs/4919831435?check_suite_focus=true Please run mentioned script and commit changes and I will merge it :)

Comment (Samridhi-98) [reply src/components/Sidebar.tsx]: Yeah, I have fixed it please review :)

**Comment by magicmatatjahu** [comment]
@Samridhi-98 I will change prefix for that PR to `fix:` because of course there are additions, but we should treat this as missed additions :)

**Review by magicmatatjahu — APPROVED** [review]
LGTM!

**Pull request merged**